ci.nix

```diff
@@ -97,19 +97,17 @@ rec {
 
             # in non-release mode collect all *.dump-hi files (required for weeder)
             postInstall = if release then null else weeder-hacks.collect-dump-hi-files;
+
+            preBuild = ''
+              export MORLEY_DOC_GIT_COMMIT_SHA=${if release then pkgs.lib.escapeShellArg commitSha else "UNSPECIFIED"}
+              export MORLEY_DOC_GIT_COMMIT_DATE=${if release then pkgs.lib.escapeShellArg commitDate else "UNSPECIFIED"}
+            '';
```

Comment:
:thinking: given that these are selected using env variables, couldn't we remove this `preBuild` and instead set the variables in the pipeline itself (as we do for cleveland options)?

Reply:
nix-build won't inherit environment from the interactive shell (or at least it shouldn't). In fact, it shouldn't inherit environment from anywhere, because that would jeopardize the whole "reproducible builds" idea.

Reply:
oh, of course :facepalm: 

For a moment I forgot that these aren't runtime options, never mind :ok_hand: